index.html

```diff
@@ -0,0 +1,205 @@
+            <img src="/svg/more.svg" class="sidebar-menu-items-icons" />
+            More
+          </div>
+          <button class="sidebar-tweet">Tweet</button>
+        </div>
+      </div>
+      <!-- Left side-bar End -->
+
+      <!-- Main Starts -->
+      <div class="main">
+        <div class="tweet">
+          <img class= "tweet-author" src="/images/profile-image.JPG" height="200px">
+        <div class="tweet-main">
+          <div class="tweet-header">
+            <div class="tweet-name">
+              Gideon Usenbor
+            </div>
+            <div class="tweet-user">
+              @gideonnn
+            </div>
+            <div class="tweet-time">
+              50m
+            </div>
+          </div>
+          <div class="tweet-content">
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit. Voluptate ut ullam, ipsam aspernatur rerum eos mollitia minima dolore maiores. Quod unde facilis a possimus quo non consequatur! Aperiam, animi reiciendis!
+          </div>
+        </div>
+      </div>
+      <div class="main">
+        <div class="tweet">
+          <img class= "tweet-author" src="/images/profile-image.JPG" height="200px">
+        <div class="tweet-main">
+          <div class="tweet-header">
+            <div class="tweet-name">
+              Gideon Usenbor
+            </div>
+            <div class="tweet-user">
+              @gideonnn
+            </div>
+            <div class="tweet-time">
+              50m
+            </div>
+          </div>
+          <div class="tweet-content">
+            Lorem ipsum dolor sit amet consectetur, adipisicing elit. Eius iusto reprehenderit quam cumque vel ut quos provident architecto deserunt iste distinctio, sed asperiores pariatur, earum placeat illo veniam. Voluptatem, repellat.
+            <img src="" alt="">
+          </div>
+        </div>
+      </div>
+      </div>    <div class="main">
+        <div class="tweet">
+          <img class= "tweet-author" src="/images/profile-image.JPG" height="200px">
+        <div class="tweet-main">
+          <div class="tweet-header">
+            <div class="tweet-name">
+              Gideon Usenbor
+            </div>
+            <div class="tweet-user">
+              @gideonnn
+            </div>
+            <div class="tweet-time">
+              50m
+            </div>
+          </div>
+          <div class="tweet-content">
+            Lorem ipsum dolor sit, amet consectetur adipisicing elit. Culpa sint laborum repellat sed quod illum ipsum consequuntur placeat aliquid, veritatis ut, cupiditate quae quo voluptatum! Tempore adipisci inventore quibusdam doloribus.
+          </div>
+        </div>
+      </div>
+      </div>    <div class="main">
+        <div class="tweet">
+          <img class= "tweet-author" src="/images/profile-image.JPG" height="200px">
+        <div class="tweet-main">
+          <div class="tweet-header">
+            <div class="tweet-name">
```

Comment:
love your use of class names having tweet-(element), makes it easier in the CSS to know which part it belongs to!